docs/minikube.md

```diff
@@ -0,0 +1,43 @@
+# Deploying Antrea on Minikube
+
+<!-- toc -->
+- [Install Minikube to your system](#install-minikube-to-your-system)
+  - [Deploy Antrea to your Minikube cluster](#deploy-antrea-to-your-minikube-cluster)
+  - [Deploy a local build of Antrea to your Minikube cluster (for developers)](#deploy-a-local-build-of-antrea-to-your-minikube-cluster-for-developers)
+  - [Check that everything is working](#check-that-everything-is-working)
+<!-- /toc -->
+
+### Install Minikube to your system
+Follow these [steps](https://minikube.sigs.k8s.io/docs/start) to install minikube and set its development environment.
+
+### Deploy Antrea to your Minikube cluster
+
+```bash
+# curl is required because --cni flag, does not accept URL as a parameter
+curl -Lo https://github.com/antrea-io/antrea/releases/download/<TAG>/antrea.yml
+minikube start --cni=antrea.yml --network-plugin=cni
+```
+
+### Deploy a local build of Antrea to your Minikube cluster (for developers)
+
+These instructions assume that you have built the Antrea Docker image locally
+(e.g. by running `make` from the root of the repository, or in case of arm64 architecture by running
+`DOCKER_BUILDKIT=1 ./hack/build-antrea-ubuntu-all.sh --platform linux/arm64`).
+
+```bash
+# load the Antrea Docker image in the minikube nodes
+minikube image load projects.registry.vmware.com/antrea/antrea-ubuntu:latest
+# deploy Antrea
+kubectl apply -f /antrea/build/yamls/antrea.yml
+```
+
+### Check that everything is working
+
+After a few seconds you should be able to observe the following when running
+`kubectl get -n kube-system pods -l app=antrea`:
```

Comment:
even though this command is working, I would suggest to change it to following one:
```
kubectl get pods -l app=antrea -n kube-system
```